src/main/config/firecloud-orchestration-compose.yaml.ctmpl

```diff
@@ -15,6 +15,12 @@ app:
     - FIRECLOUD_URL_ROOT={{.Data.env_firecloud_url_root}}
     - GOOGLE_SECRET_JSON={{.Data.env_google_secret_json}}
     - ADMIN_REFRESH_TOKEN={{.Data.env_admin_refresh_token}}
+    - ORCHESTRATION_PEM=/etc/firecloud-account.pem
+    {{$pem_file := printf "secret/dsde/%s/common/firecloud-account.pem" $environment}}
+    {{with vault $pem_file}}
+    - ORCHESTRATION_PEM_CLIENT_ID={{.Data.client_id}}
+    {{end}}
+    - NIH_WHITELIST_BUCKET={{.Data.env_nih_whitelist_bucket}}
```

Comment:
The PEM stuff *is* in Vault.  AFAIK we have not decided on a proper bucket name yet.